node_io/lib/src/stdin.dart

```diff
@@ -17,6 +17,9 @@ class Stdin extends ReadableStream<List<int>> implements io.Stdin {
   @override
   late bool echoMode;
 
+  @override
+  late bool echoNewlineMode;
```

Comment:
Just having a `late` member that doesn't ever get set seems wrong. It would be better to explicitly throw an `UnimplementedError` if we can't support this.

Reply:
Done